.github/workflows/benchmark_execution.yml

```diff
@@ -122,8 +132,15 @@ jobs:
           mkdir -p "${E2E_TEST_ARTIFACTS_DIR}"
           jq -r \
             --arg DEVICE_NAME "${DEVICE_NAME}" \
+            --arg SHARD_INDEX "${SHARD_INDEX}" \
+            --arg GCS_ARTIFACT_DIR "${E2E_TEST_ARTIFACTS_GCS_ARTIFACT_DIR}" \
+            '.[$DEVICE_NAME].shards[] | select((.shard_index | tostring) == $SHARD_INDEX) | .module_dir_paths[] | "\($GCS_ARTIFACT_DIR)/\(.)"' \
+            "${BENCHMARK_CONFIG}"
```

Comment:
Is this part duplicated below? Also looks like the results is not using.

Reply:
Oh yeah. This is only for debugging and I forgot to remove it. Thanks for spotting.